athena-query-execution-pipeline/requirements.txt

```diff
@@ -0,0 +1,6 @@
+aws-cdk-lib==2.54.0
```

Comment:
Do you want to just bump this to `2.61.0` while we're at it?

Reply:
Done !